client/utils/guidance.py

```diff
@@ -93,32 +120,13 @@ def show_guidance_msg(self, unique_id, input_lines, access_token, hash_key, guid
             print(GUIDANCE_DEFAULT_MSG)
             return EMPTY_MISUCOUNT_TGID_PRNTEDMSG
 
-        shorten_unique_id = assess_id_util.canonicalize(unique_id)
-        # Try to get the info dictionary for this question. Maps wrong answer
-        # to dictionary
-        Wrong_Answer_2_dict_info = self.guidance_json[
-            'dictAssessId2WA2DictInfo'].get(shorten_unique_id)
-
-        # get returns None if Wrong_Answer_2_dict_info doesn't have
-        # shorten_unique_id in dictionary
-        if not Wrong_Answer_2_dict_info:
-            log.info("shorten_unique_id is not in Wrong_Answer_2_dict_info")
-            print(GUIDANCE_DEFAULT_MSG)
-            return EMPTY_MISUCOUNT_TGID_PRNTEDMSG
+        response = repr(input_lines)
+        self.set_tg(access_token)
+        log.info("Guidance TG is %d", self.tg_id)
 
-        dict_info = Wrong_Answer_2_dict_info.get(repr(input_lines))
-
-        # If this wrong answer is not in the JSON file, display default message
-        if not dict_info:
-            log.info("Answer not in the JSON file.")
-            print(GUIDANCE_DEFAULT_MSG)
-            return EMPTY_MISUCOUNT_TGID_PRNTEDMSG
-
-        self.set_tg(access_token, guidance_flag)
         if self.tg_id == TG_ERROR_VALUE:
-            # If self.tg_id == -1, some errors happen when trying to access
-            # server
-            log.warning("Error when trying to access server.")
+            # If self.tg_id == -1, there was an error when trying to access the server
+            log.warning("Error when trying to access server. TG == -1")
             print(GUIDANCE_DEFAULT_MSG)
             return EMPTY_MISUCOUNT_TGID_PRNTEDMSG
```

Comment:
After this point shouldn't we never use `EMPTY_MISUCOUNT_TGID_PRNTEDMSG` again?

What about instead at the top of `show_guidance_msg` there's a variable like `return_val = ({}, -1, "")` and this gets slowly updated and returned as appropriate?

Also new change request:
- [ ] Please change printed_msg to be a list of printed messages, not a never end string. This would also mean `return_val = ({}, -1, [])`

And what do you think of adding a divider between each message? Like ` ~*~*~` or something.